whistle/managers.py

```diff
@@ -73,10 +74,10 @@ def notify(request, recipient, event, actor=None, object=None, target=None):
         registered_for_email = NoticeManager.is_notice_allowed(recipient, 'mail', event)
 
         if registered_for_email:
-            EmailManager.send_mail(request, recipient, event, actor, object, target)
+            EmailManager.send_mail(request, recipient, event, actor, object, target, details)
 
     @staticmethod
-    def get_description(event, actor, object, target, pass_variables=True):
+    def get_description(event, actor, object, target, details, pass_variables=True):
```

Comment:
"details" is a textfield which can contain a very long value/string. I would rather expect use notification.details where needed, not in this method. get_description() method is used in "__str__()" which shouldn't be very long.